ImageMagick/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Quick script to run ldd against binaries on a variety of platforms. 
+
+FROM ubuntu:16.04
+COPY ./Q8/lib/Release/CrossPlatform/ubuntu.16.04/Magick.NET-Q8-x64.Native.dll.so ./Magick.NET-Q8-x64.Native.dll.so
```

Comment:
I wonder if we should rename the `ubuntu.16.04` folder to something else because we use it with multiple distros.